simulation/simulation_interface/include/simulation_interface/zmq_multi_client.hpp

```diff
@@ -79,6 +79,12 @@ class MultiClient
   auto call(const simulation_api_schema::FollowPolylineTrajectoryRequest &)
     -> simulation_api_schema::FollowPolylineTrajectoryResponse;
 
+  auto call(const simulation_api_schema::SetVelocityLimitRequest &)
+    -> simulation_api_schema::SetVelocityLimitResponse;
+
+  auto call(const simulation_api_schema::SetTargetSpeedRequest &)
+    -> simulation_api_schema::SetTargetSpeedResponse;
+
```

Comment:
You're right, I intended to send this together with `FollowTrajectoryAction` however `target_speed` and `max_speed` are set by separate requests. 

`requestSpeedChange` is a separate request that comes from the interpreter (from `SpeedActionTarget` -> `applySpeedAction`). `setVelocityLimit` is a separate request that comes from the interpreter (from action `AssignControllerAction` -> `applyAssignControllerAction`). 

For this reason, setting `FollowTrajectoryAction` together with `target_speed` and `max_speed` during a single `requestFollowTrajectory` call might be difficult - it would require storing `target_speed` and `max_speed` in the `API` (of course, if `requestSpeedChange`, `setVelocityLimit` are always called before a `requestFollowTrajectory` request and cannot be called after it)

Reply:
Understood the background of the change.

However, I still would like to see a way to send the data together with the requestFollowTrajectory as much as possible for the reasons I mentioned earlier. If this requires changing the store of the necessary information as a data member of EgoEntity, then I am fine with doing that.